Project/Project/Pages/Index.cshtml.cs

```diff
@@ -68,16 +71,9 @@ public IActionResult OnGet(string query)
                 {
                     SearchCompleted = true;
                 }
-                else
```

Comment:
Removed the else condition where it said SearchCompleted = false, since this statement was added outside the loop in the beginning to reduce number of lines of code.